pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm not clear how this file gets used.  Each of the applications is self-contained and has its own `pom.xml` file in addition to `.platform.app.yaml`,  no?

Reply:
This file is to show applications available using `mvn clean`, furthermore, to we use it on the PRs.
Therefore, when I update the library and update the version. It'll compile app applications to make a green build in the PR.